src/components/Footer/Footer.tsx

```diff
@@ -13,11 +13,13 @@ const Footer = () => {
         }, {
             name: "Perceptual Modes"
         }, {
-            name: "IDEO"
+            name: "Ideo"
         }, {
             name: "IT"
         }, {
             name: "Dormant"
+        }, {
+            name: "NCIS"
         }]
```

Comment:
It would probably be best to leave dormant at the very end since it's just a placeholder 😁 

```suggestion
            name: "NCIS"
        }, {
            name: "Dormant"
        }]
```